dump/data-1-documents_page.sql

```diff
@@ -1,66 +1,66 @@
-INSERT INTO documents_page (`id`,`controller`,`template`,`title`,`description`,`metaData`,`prettyUrl`,`contentMasterDocumentId`,`targetGroupIds`,`missingRequiredEditable`) VALUES ('200','App\\Controller\\ContentController::defaultAction','default/events.html.twig','','','a:0:{}',NULL,NULL,'',NULL);
+INSERT INTO documents_page (`id`,`controller`,`template`,`title`,`description`,`metaData`,`prettyUrl`,`contentMasterDocumentId`,`targetGroupIds`,`missingRequiredEditable`,`staticGeneratorEnabled`) VALUES ('1','App\\Controller\\ContentController::portalAction',NULL,'','','a:0:{}',NULL,NULL,'',NULL,NULL);
+INSERT INTO documents_page (`id`,`controller`,`template`,`title`,`description`,`metaData`,`prettyUrl`,`contentMasterDocumentId`,`targetGroupIds`,`missingRequiredEditable`,`staticGeneratorEnabled`) VALUES ('3','App\\Controller\\DefaultController::defaultAction',NULL,'','','a:0:{}',NULL,NULL,'',NULL,NULL);
+INSERT INTO documents_page (`id`,`controller`,`template`,`title`,`description`,`metaData`,`prettyUrl`,`contentMasterDocumentId`,`targetGroupIds`,`missingRequiredEditable`,`staticGeneratorEnabled`) VALUES ('12','App\\Controller\\DefaultController::defaultAction','includes/footer.html.twig','Pimcore Footer','','a:0:{}',NULL,NULL,'',NULL,NULL);
+INSERT INTO documents_page (`id`,`controller`,`template`,`title`,`description`,`metaData`,`prettyUrl`,`contentMasterDocumentId`,`targetGroupIds`,`missingRequiredEditable`,`staticGeneratorEnabled`) VALUES ('14','App\\Controller\\DefaultController::defaultAction',NULL,'','','a:0:{}',NULL,NULL,'',NULL,NULL);
+INSERT INTO documents_page (`id`,`controller`,`template`,`title`,`description`,`metaData`,`prettyUrl`,`contentMasterDocumentId`,`targetGroupIds`,`missingRequiredEditable`,`staticGeneratorEnabled`) VALUES ('38','App\\Controller\\ContentController::editableRoundupAction',NULL,'Editable Roundup','','a:0:{}',NULL,NULL,'',NULL,NULL);
+INSERT INTO documents_page (`id`,`controller`,`template`,`title`,`description`,`metaData`,`prettyUrl`,`contentMasterDocumentId`,`targetGroupIds`,`missingRequiredEditable`,`staticGeneratorEnabled`) VALUES ('39','App\\Controller\\ContentController::thumbnailsAction',NULL,'Thumbnails','','a:0:{}',NULL,NULL,'',NULL,NULL);
+INSERT INTO documents_page (`id`,`controller`,`template`,`title`,`description`,`metaData`,`prettyUrl`,`contentMasterDocumentId`,`targetGroupIds`,`missingRequiredEditable`,`staticGeneratorEnabled`) VALUES ('45','App\\Controller\\ContentController::defaultAction',NULL,'','','a:0:{}',NULL,NULL,'',NULL,NULL);
+INSERT INTO documents_page (`id`,`controller`,`template`,`title`,`description`,`metaData`,`prettyUrl`,`contentMasterDocumentId`,`targetGroupIds`,`missingRequiredEditable`,`staticGeneratorEnabled`) VALUES ('46','App\\Controller\\ContentController::defaultAction',NULL,'','','a:0:{}',NULL,NULL,'',NULL,1);
```

Comment:
Please mark [Terms-and-Conditions](https://demo.pimcore.fun/en/More-Stuff/Terms-and-Conditions) page as static instead of Magazine.